uxarray/dataset.py

```diff
@@ -0,0 +1,29 @@
+"""uxarray dataset module."""
+
+from logging import raiseExceptions
```

Comment:
From line 3 through 7, the imported packages are unused. Should they be removed? Just wanted to double check that there isn't a technical reason for them to be here